src/app/complex/complex-details/complex-references/external-resources/external-resources.component.ts

```diff
@@ -7,23 +7,41 @@ import {humapSearchUrl} from '../../../complex-portal-utils';
   templateUrl: './external-resources.component.html',
   styleUrls: ['./external-resources.component.scss']
 })
-export class ExternalResourcesComponent implements OnInit {
-  private _crossReferences: CrossReference[];
+export class ExternalResourcesComponent {
 
   protected readonly humapSearchUrl = humapSearchUrl;
 
+  crossReferences = input<CrossReference[]>();
+  externalResources = input<CrossReference[]>();
+  institution = input<string>();
+
+  nDexBioDomain: string;
+  nDexBioNetwork: string;
+
   constructor() {
+    effect(() => this.findNDexBioDomainAndNetwork(this.institution(), this.crossReferences()));
   }
 
-  ngOnInit() {
+  nDexBioSearchUrl(id: string): string {
+    return this.nDexBioDomain + '/viewer/networks/' + this.nDexBioNetwork + '?query=' + id + '&queryType=twoStepAdjacent';
   }
 
-  get crossReferences(): CrossReference[] {
-    return this._crossReferences;
-  }
+  private findNDexBioDomainAndNetwork(institution: string, crossReferences: CrossReference[]) {
+    if (institution.toLowerCase() === 'music') {
+      for (let i = 0; i < crossReferences.length; i++) {
+        const crossRef = crossReferences[i];
+        const database = crossReferences[i].database.toLowerCase();
 
-  @Input()
-  set crossReferences(value: CrossReference[]) {
-    this._crossReferences = value;
+        if (database === 'cell line ontology') {
+          if (crossRef.identifier === 'CLO:0037372') {
+            this.nDexBioDomain = 'https://www.ndexbio.org';
```

Comment:
What if you have both terms in the crossReferences? The second one would overwrite the first one, is that desired?

Reply:
This should be a temporary fix. At the moment we cannot have both, and hopefully this is removed before we ended up in that scenario.